distribution/ddf-common/src/main/resources-filtered/etc/org.apache.karaf.features.xml

```diff
@@ -43,7 +45,7 @@
                 and felix.scr tries to use R7 logging even though the current version of pax-logging
                 does not support it. This override can be removed when pax-logging supports R7
                 (support was added in v2.0.1. See https://ops4j1.jira.com/browse/PAXLOGGING-255). -->
-        <bundle originalUri="mvn:org.apache.felix/org.apache.felix.scr/2.1.20"
+        <bundle originalUri="mvn:org.apache.felix/org.apache.felix.scr/2.1.24"
```

Comment:
Based on the comment here, we should be able to remove this bundle replacement now that we're on pax-logging 2.0.6